pkg/controller/certificates/internal/secretsmanager/keystore_test.go

```diff
@@ -23,13 +23,12 @@ import (
 	"fmt"
 	"testing"
 
-	jks "github.com/pavel-v-chernykh/keystore-go"
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	"github.com/jetstack/cert-manager/pkg/util/pki"
+	jks "github.com/pavel-v-chernykh/keystore-go/v4"
```

Comment:
Please use the [cert-manager import format](https://cert-manager.io/docs/contributing/coding-conventions/#organizing-imports).